apps/remix-ide/src/app.js

```diff
@@ -347,6 +349,34 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
     settings
   ])
 
+  if (!registry.get('config').api.exists('settings/matomo-analytics')) {
+  // Ask to opt in to Matomo for remix, remix-alpha and remix-beta
+  //if (window.location.hostname.includes('.ethereum.org')) {
+    modalDialog(
+      'Help us to improve our IDE!',
+      yo`<div>
+        >p>Remix IDE uses <a href="https://matomo.org">Matomo</a>, an open source data analytics software to improve the use of our website.</p>
+        <p>All data collected through Matomo is stored at our own server - no data is given to third parties.</p>
+        <p>We do not store any personally identifiable information (PII).</p>
+        <p>You can change your choice in the Settings panel.</p>
+      </div>`,
+      {
+        label: "Sure",
+        fn: () => {
+          settings.updateMatomoAnalyticsChoice(true)
+          _paq.push(['forgetUserOptOut'])
+        }
+      },
+      {
+        label: 'Decline',
+        fn: () => {
+          settings.updateMatomoAnalyticsChoice(false)
```

Comment:
that is not how should be done an API call, and surely that shouldn't be put in the remix plugin API